### DISCUSSION TITLE
Welcome to DotNetSiemensPLCToolBoxLibrary Discussions!

### DISCUSSION THREAD

**Discussion by jogibear9988** [dotnetprojects/DotNetSiemensPLCToolBoxLibrary#159] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 99124zch** [comment]
您好！请问一下，下面这个报错咋解决，困扰我们好久了，望回复，谢谢！

System.IO.FileNotFoundException:“未能加载文件或程序集“Siemens.Engineering, Version=15.1.0.0, Culture=neutral, PublicKeyToken=d29ec89bac048f84”或它的某一个依赖项。系统找不到指定的文件。”

**Comment by jonmoney** [comment]
Hello, I am writing a tool to browse Siemens TIA Portal V18 projects. I am wanting to retrieve the individual array element comments from the arrays. Is this possible? So far, all I can see is the array in the DB. I am keeping my array data in the "r" datablock.
